CreateItem/CreateItemModal.js

```diff
@@ -256,7 +255,7 @@ export default class CreateItemModal extends React.Component {
               <FormattedMessage id={`ui-plugin-create-item.error.${errorKey}`} />
             </div>
           )}
-          <this.connectedApp
+          <ItemForm
             {...props}
```

Comment:
I would suggest to replace here `{...props}` with `stripes={stripes}`

Reply:
`ItemForm` requires not only stripes

Reply:
like what? the thing that it's hidden and could cause problems

Reply:
`okapi`